tyr/alerts/stackdriver.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf8 -*-
+import requests
+import json
+import logging
+import re
+import os
+
+
+API_ENDPOINT = 'https://api.stackdriver.com/'
+API_KEY = os.environ['STACKDRIVER_API_KEY']
+LOGIN_URL = 'https://app.stackdriver.com/account/login/'
+POST_ALERT_URL = "https://app.stackdriver.com/api/alerting/policy"
+
+log = logging.getLogger('tyr.alerts.StackDriver')
+if not log.handlers:
+    log.setLevel(logging.INFO)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    formatter = logging.Formatter(
+        '%(asctime)s [%(name)s] %(levelname)s: %(message)s',
+        datefmt='%H:%M:%S')
+    ch.setFormatter(formatter)
+    log.addHandler(ch)
+
+LIST_POLICY_URL = "/v0.2/alerting/policies/"
+LIST_GROUPS_URL = "/v0.2/groups/"
+GET_POLICY_URL = "/v0.2/alerting/policies/{policy_id}/"
+GET_GROUP_URL = "/v0.2/groups/{group_id}/"
+LIST_GROUP_MEMBERS_URL = "/v0.2/groups/{group_id}/members/"
+
+
+class StackDriver:
+
+    def __init__(self):
+        self.headers = {"x-stackdriver-apikey": API_KEY}
+        self.read_config()
+        self.policies = self.get_policy_list()
+        self.groups = self.get_group_list()
+
+    def exit_flush(self, code):
+        logging.shutdown()
+        exit(code)
+
+    def read_config(self):
+        with open(os.path.abspath(os.path.dirname(__file__)) +
+                  '/../policies/stackdriver_conditions.json', 'r') as jsonfile:
+            self.conditions = json.load(jsonfile)
+
+        with open(os.path.abspath(os.path.dirname(__file__)) +
+                  '/../policies/stackdriver_notifications.json',
+                  'r') as jsonfile:
+            self.notifications = json.load(jsonfile)
+
+    def get_policy_list(self):
+        r = requests.get(API_ENDPOINT + LIST_POLICY_URL, headers=self.headers)
```

Comment:
You should probably check the status code to make sure the request was successful.